src/style.css

```diff
@@ -1 +1,127 @@
-/* empty source */
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 32px 16px 16px ;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+}
+
+.card__iMac {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__discription {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
```

Comment:
do fix the size because it should be stretched the full width of a card and the height should depend on the font size

If you need more spacing just use margin-top